errors/errors.go

```diff
@@ -36,4 +36,5 @@ var (
 	ErrIllegalConfigKey        = errors.New("cli: given config key is not allowed")
 	ErrScanNotSupported        = errors.New("search: scanning of image media type not supported")
 	ErrCLITimeout              = errors.New("cli: Query timed out while waiting for results")
+	ErrConfigNameUsed          = errors.New("config: config name is used. see 'zot config -l'")
```

Comment:
ErrDuplicateConfigName = errors.New("cli: cli config name already added. see 'zot config -l'") ??

Reply:
Would leave out the 'see 'zot config -l' from the error. Catch/check for this error in code and then emit that remedial msg.